3.7/graphs-satellite-graphs.md

```diff
@@ -12,116 +12,120 @@ SatelliteGraphs are only available in the
-    @endDocuBlock satelliteGraphExplain1_cluster
-{% endarangoshexample %}
-{% include arangoshexample.html id=examplevar script=script result=result %}
+However, some assumptions have to be made:
+* The graph size should be manageable.
+* The performance will be the highest if you're not permanently updating the
+  graph's structure and content.
 
-Let us analyze a query involving a traversal:
+When doing joins involving graph traversals, shortest path or k-shortest paths
+computation in an ArangoDB cluster, data has to be exchanged between different
+servers. In particular graph traversals are usually executed on a Coordinator,
+because they need global information. This results in a lot of network traffic
+and potentially slow query execution.
 
-{% arangoshexample examplevar="examplevar" script="script" result="result" %}
-    @startDocuBlockInline satelliteGraphExplain2_cluster
-    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphExplain2_cluster}
-    ~var graphModule = require("@arangodb/general-graph");
-    ~var satelliteGraphModule = require("@arangodb/satellite-graph");
-    ~graphModule._create("normalGraph", [ graphModule._relation("edges", "vertices", "vertices") ], [], {});
-    ~satelliteGraphModule._create("satelliteGraph", [ satelliteGraphModule._relation("satEdges", "satVertices", "satVertices") ], [], {});
-    ~db._create("collection", {numberOfShards: 8});
-    db._explain(`FOR doc in collection FOR v,e,p IN OUTBOUND "vertices/start" GRAPH "normalGraph" RETURN [doc,v,e,p]`, {}, {colors: false});
-    ~db._drop("collection");
-    ~satelliteGraphModule._drop("satelliteGraph", true);
-    ~graphModule._drop("normalGraph", true);
-    @END_EXAMPLE_ARANGOSH_OUTPUT
-    @endDocuBlock satelliteGraphExplain2_cluster
-{% endarangoshexample %}
-{% include arangoshexample.html id=examplevar script=script result=result %}
+A short example:
+
+Take a graph-based permissions use case where you have a large, sharded collection
+of documents within your ArangoDB cluster. They don't need to be part of your graph.
+You probably want to determine quickly, if a user, group or device has permission to
+e.g. access certain information. With SatelliteGraphs you can now replicate your
+graph handling the permissions to each DB-Server and execute queries locally.
+
+
+When to use another graph type?
+--------------
+
+Due the fact that we need to replicate graph data to the participating DB-Servers,
+performance of writes into the affected graph collections will become slower. Also,
+as writes are performed on every DB-Server, you will allocate more storage
+distributed around the whole cluster environment.
+
+If you want to distribute a very large graph and you don't want to replicate all
+graph data to all participating nodes that are part of your cluster. In that case
+please take a look at [SmartGraphs](graphs-smart-graphs.html)
+
+Getting started
+--------------
+
+First of all, compared to GeneralGraphs, SatelliteGraphs do have  some rules regarding
```

Comment:
As opposed to my comment above here you write `GeneralGraphs` without a gap, I would like a consistent writing, I do not care too much which of the writings it is.

Reply:
@Simran-B what do we want to use? GeneralGraphs or General Graphs?

Reply:
General Graphs as per Ingo